tests/runtime/events.ts

```diff
@@ -0,0 +1,61 @@
+import { Event, EventParam } from './ethereum'
+import { hash, addr } from './hex'
+
+export const DEFAULT_CONTRACT_ADDRESS = addr('0x0001020304050607080910111213141516171819')
```

Comment:
Why is this different from the 0xff..ff default address below?

Reply:
This gets addressed in a later PR.